core/trino-main/src/test/java/io/trino/sql/planner/TestInsert.java

```diff
@@ -180,24 +180,22 @@ public void testCreateTableAsSelectUnpartitioned()
     {
         assertDistributedPlan(
                 "CREATE TABLE new_test_table_unpartitioned (column1, column2) AS SELECT * FROM (VALUES (1, 2)) t(column1, column2)",
-                withoutPreferredPartitioning(),
+                defaultSession(),
                 anyTree(
                         node(TableWriterNode.class,
                                 // round robin
                                 exchange(LOCAL, REPARTITION, ImmutableList.of(), ImmutableSet.of(),
                                         values("column1", "column2")))));
     }
 
-    private Session withPreferredPartitioning()
+    private Session withPreferredPartitioning(Session session)
     {
-        return Session.builder(getQueryRunner().getDefaultSession())
+        return Session.builder(session)
                 .setSystemProperty(USE_PREFERRED_WRITE_PARTITIONING, "true")
-                .setSystemProperty(REDISTRIBUTE_WRITES, "false")
-                .setSystemProperty(TASK_WRITER_COUNT, "16")
                 .build();
     }
 
-    private Session withoutPreferredPartitioning()
```

Comment:
Let's keep explicit session as it was. It's easier for reader to understand what's going on.